CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 # Changelog
 
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
```

Comment:
@bastelfreak Do camptocamp actually use modulesync for this module though? :)

Reply:
Good point. I think we stopped using it a few months ago.